_includes/ref/bitcoin-core/rpcs/intro.md

```diff
@@ -9,153 +9,199 @@ http://opensource.org/licenses/MIT.
 
 {% autocrossref %}
 
-Bitcoin Core provides a large number of Remote Procedure Calls (RPCs)
-using a [HTTP JSON-RPC version 1.0][] interface. Any program can access
-the RPCs using JSON-RPC, but Bitcoin Core also provides the
-`bitcoin-cli` command to wrap the JSON-RPC access for Command Line
-Interface (CLI) users. Most of the RPC examples in this documentation
-use `bitcoin-cli` for simplicity, so this subsection describes raw
-JSON-RPC interface and how the command-line interface translates it.
-
-In order to start `bitcoind`, you will need to set a password for
-JSON-RPC in the `bitcoin.conf` file. See the [Examples
-Page][devexamples] for details. JSON-RPC starts on port 8332 for mainnet
-and 18332 for testnet and regtest. By default, `bitcoind` doesn't use a
-JSON-RPC user, but you can set one (see `bitcoind --help`).
-
-The interface is not intended for public access and is only accessible
-from localhost by default.
-
-RPCs are made using the standard JSON-RPC 1.0 syntax, which sends several
-standard arguments:
+Bitcoin Core provides a remote procedure call (RPC) interface for various
+administrative tasks, wallet operations, and queries about network and block
+chain data.
+
+If you start Bitcoin Core using `bitcoin-xt`, the RPC interface is disabled by
```

Comment:
Indeed I did. Good eye. What a difference a letter makes :) I've fixed this locally on my laptop. Unless somebody tells me otherwise, I'll bundle all these feedback-inspired changes into a single commit.